spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -156,6 +164,8 @@
 })
 @EnableJpaAuditing
 @EnableTransactionManagement
+@EnableAsync
+@EnableScheduling
```

Comment:
I believe both these annotatios are not needed here anymore.

Reply:
Good catch! will remove these.